mono_install.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Minio .NET Library for Amazon S3 Compatible Cloud Storage, (C) 2017 Minio, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Download and install Mono and .NETCore for Ubuntu 16.04
+main() {
+	sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ yakkety main" > /etc/apt/sources.list.d/dotnetdev.list' 
```

Comment:
Ubuntu 16.04 is Xenial, not Yakkety. This is why dotnet wouldn't install for me when I use mono_install.sh. I see the following error,
```
➜  minio-dotnet git:(r149) sudo apt-get install -yq dotnet-dev-1.0.4
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 dotnet-dev-1.0.4 : Depends: libstdc++6 (>= 6) but 5.4.0-6ubuntu1~16.04.4 is to be installed
                    Depends: dotnet-sharedframework-microsoft.netcore.app-1.1.2 but it is not going to be installed
E: Unable to correct problems, you have held broken packages.

```